packages/create-block-interactive-template/block-templates/edit.js.mustache

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress components that create the necessary UI elements for the block
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-components/
+ */
+import { TextControl } from '@wordpress/components';
+
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#useblockprops
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * The edit function describes the structure of your block in the context of the
+ * editor. This represents what the editor will render when the block is used.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#edit
+ *
+ * @param {Object}   props               Properties passed to the function.
+ * @param {Object}   props.attributes    Available block attributes.
+ * @param {Function} props.setAttributes Function that updates individual attributes.
+ *
+ * @return {WPElement} Element to render.
+ */
+export default function Edit( { attributes, setAttributes } ) {
+	const blockProps = useBlockProps();
+	return (
+		<div { ...blockProps }>
+			<TextControl
+				value={ attributes.message }
+				onChange={ ( val ) => setAttributes( { message: val } ) }
+			/>
```

Comment:
You are saving a `message` attribute here, but it's not declared in the `block.json` or used in the `render.php`. 

If you want to keep it the attribute, you should probably add it to the `block.json` and use it `render.php` to make the example complete. If you prefer not to use it, you can just output a [static text like this](https://github.com/WordPress/gutenberg/blob/trunk/packages/create-block/lib/templates/block/edit.js.mustache#L35) instead.

Reply:
Thanks for the feedback here. 
I've simplified this mustache template based on [the example you suggest](https://github.com/WordPress/gutenberg/blob/trunk/packages/create-block/lib/templates/block/edit.js.mustache#L35) 👍